core/domain/role_domain.py

```diff
@@ -0,0 +1,76 @@
+# coding: utf-8
```

Comment:
oh ! The file 'role_services_test' existed and I was running 'core.domain.role_services_tests'.